repose-aggregator/src/docs/asciidoc/filters/attribute-mapping-policy-validation.adoc

```diff
@@ -0,0 +1,58 @@
+= Attribute Mapping Policy Validation Filter
+
+include::../_includes/in-progress.adoc[]
+
+This filter will validate attribute mapping policies in requests to the origin service.
+
+The origin service is expected to be an Identity service.
+This filter will only take action if the HTTP request method is `PUT`.
+
+== General filter information
+* *Name:* attribute-mapping-policy-validation
+* *Released:* TBD
```

Comment:
Yeah, when I wrote that, I was wondering how we have handled this scenario up until now. Have we really just not released any new filters since moving to versioned docs?

Anyway, I agree. I don't think the next release story exists yet, though, so I'm not sure where to track that...